common/namespace/dlqMessageHandler_test.go

```diff
@@ -296,9 +296,6 @@ func (s *dlqMessageHandlerSuite) TestMergeMessages_ThrowErrorOnHandleReceivingTa
 		Return(tasks, nil, nil)
 	s.mockReplicationTaskExecutor.EXPECT().Execute(namespaceAttribute1).Return(nil)
 	s.mockReplicationTaskExecutor.EXPECT().Execute(namespaceAttribute2).Return(testError)
-	s.mockReplicationQueue.EXPECT().DeleteMessageFromDLQ(messageID1).Return(nil)
-	s.mockReplicationQueue.EXPECT().DeleteMessageFromDLQ(messageID2).Times(0)
-	s.mockReplicationQueue.EXPECT().UpdateDLQAckLevel(messageID1).Return(nil)
```

Comment:
After update gomock these tests started to fail and it turned out these were never called but previous version of gomock ignored it due some bug, I believe.